.github/workflows/backend-integration-tests.yml

```
@@ -1,82 +1,82 @@
+        env:
+          NODE_ENV: test
+          DB_PATH: /tmp/stellar-goal-vault-test.db
+      - name: Upload backend coverage HTML
+        if: always()
+        uses: actions/upload-artifact@v4
+        with:
+          name: backend-coverage-html
+          path: backend/coverage
+
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.12)

[error] the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)